File: python/grammer/17_function.py
"""17) 함수: 입력값을 받아 결과를 반환하는 구조."""

def gcd(a: int, b: int) -> int:
    """유클리드 호제법으로 최대공약수 계산."""
    while b:
        a, b = b, a % b
    return a

x, y = 12, 18
print(f"gcd({x}, {y}) =", gcd(x, y))

File: cpp/grammer/17_function.cpp
#include <iostream>

int gcd(int a, int b) {
    while (b) {
        int t = a % b;
        a = b;
        b = t;
    }
    return a;
}

int main() {
    int x = 12, y = 18;
    std::cout << "gcd(" << x << ", " << y << ") = " << gcd(x, y) << "\n";
}
